src/main/java/roomescape/respository/ThemeDAO.java

```diff
@@ -0,0 +1,63 @@
+package roomescape.respository;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import roomescape.model.Theme;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+
+@Repository
+public class ThemeDAO {
+    private final JdbcTemplate jdbcTemplate;
+    private final RowMapper<Theme> rowMapper = (resultSet, rowNumber) -> {
+        Theme theme = new Theme(
+                resultSet.getLong("id"),
+                resultSet.getString("name"),
+                resultSet.getString("description"),
+                resultSet.getString("thumbnail")
+        );
+        return theme;
```

Comment:
```suggestion
        return new Theme(
                resultSet.getLong("id"),
                resultSet.getString("name"),
                resultSet.getString("description"),
                resultSet.getString("thumbnail")
        );
```

위와 같이 바로 return 해볼수도 있을 것 같아요.